dpnp/dpnp_iface_trigonometric.py

```diff
@@ -663,14 +662,13 @@ def log(x1):
     [0.0, 1.0, 2.0, -inf]
 
     """
+    if use_origin_backend(x1):
+        if not isinstance(x1, dparray):
+            pass
```

Comment:
It is better to move this fixes into other PR. Or provide, please, full description for this PR.